app/main.py

```diff
@@ -1,3 +1,47 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int | float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: int | float | Distance) -> Distance:
+        if isinstance(other, Distance):
+            distance3 = self.km + other.km
+        else:
+            distance3 = self.km + other
+        return Distance(distance3)
+
+    def __iadd__(self, other: int | float | Distance) -> Distance:
+        if isinstance(other, Distance):
+            self.km += other.km
+        else:
+            self.km += other
+        return self
+
+    def __mul__(self, other: int | float) -> Distance:
+        return Distance(self.km * other)
+
+    def __truediv__(self, other: int | float) -> Distance:
+        return Distance(round(self.km / other, 2))
+
+    def __lt__(self, other: int | float) -> bool:
```

Comment:
```suggestion
    def __lt__(self, other: int | float | Distance) -> bool:
            if isinstance(other, Distance):
              return self.km < other.km
```

Reply:
Add also for all of the bellow functions if they get Distance as other

Reply:
ah, ok, thx